packages/@aws-cdk/aws-cloudfront-origins/test/integ.s3-origin-custom-headers.ts

```diff
@@ -0,0 +1,21 @@
+import * as cloudfront from '@aws-cdk/aws-cloudfront';
+import * as s3 from '@aws-cdk/aws-s3';
+import * as cdk from '@aws-cdk/core';
+import * as origins from '../lib';
+
+const app = new cdk.App();
+
+const stack = new cdk.Stack(app, 'cloudfront-s3-origin-custom-headers');
+
+const bucket = new s3.Bucket(stack, 'Bucket');
+new cloudfront.Distribution(stack, 'CustomHeadersDistribution', {
+  defaultBehavior: {
+    origin: new origins.S3Origin(bucket, {
+      customHeaders: {
```

Comment:
Distributions take a long time to deploy and tear down, so I'm a bit skeptical of adding a new integ test just for this. I'd either drop it altogether, or add it to the existing `integ.s3-origin.ts` test.

Reply:
Removed.